EIPS/eip-1.md

```diff
@@ -3,121 +3,108 @@ eip: 1
+  - **ERC** — application-level standards and conventions, including contract standards such as token standards ([ERC20]), name registries ([ERC26], [ERC137]), URI schemes ([ERC67]), library/package formats ([EIP82]), and wallet formats ([EIP75], [EIP85]).
+- An **Informational EIP** describes an Ethereum design issue, or provides general guidelines or information to the Ethereum community, but does not propose a new feature. Informational EIPs do not necessarily represent Ethereum community consensus or a recommendation, so users and implementers are free to ignore Informational EIPs or follow their advice.
+- A **Meta EIP** describes a process surrounding Ethereum or proposes a change to (or an event in) a process. Process EIPs are like Standards Track EIPs but apply to areas other than the Ethereum protocol itself. They may propose an implementation, but not to Ethereum's codebase; they often require community consensus; unlike Informational EIPs, they are more than recommendations, and users are typically not free to ignore them. Examples include procedures, guidelines, changes to the decision-making process, and changes to the tools or environment used in Ethereum development. Any meta-EIP is also considered a Process EIP.
 
--   An **Informational EIP** describes an Ethereum design issue, or provides general guidelines or information to the Ethereum community, but does not propose a new feature. Informational EIPs do not necessarily represent Ethereum community consensus or a recommendation, so users and implementers are free to ignore Informational EIPs or follow their advice.
--   A **Meta EIP** describes a process surrounding Ethereum or proposes a change to (or an event in) a process. Process EIPs are like Standards Track EIPs but apply to areas other than the Ethereum protocol itself. They may propose an implementation, but not to Ethereum's codebase; they often require community consensus; unlike Informational EIPs, they are more than recommendations, and users are typically not free to ignore them. Examples include procedures, guidelines, changes to the decision-making process, and changes to the tools or environment used in Ethereum development. Any meta-EIP is also considered a Process EIP.
+## EIP Work Flow
 
-EIP Work Flow
--------------
+The normal process is:
 
-The EIP repository editors change the EIPs status. Please send all EIP-related email to the EIP editors or preferably (e.g. for public transparency) you can ping one of the Editors that is involved with your pull request (PR); they are listed under [EIP editors below](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1.md#eip-editors). Also see [EIP Editor Responsibilities & Workflow](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1.md#eip-editor-responsibilities-and-workflow).
+```
+[ A pull request ] -> [ DRAFT ] -> [ TWO WEEK REVIEW ] -> [ FINAL ]
```

Comment:
Instead of "Two Week Review", I think "Last Call" would be a better description.

This workflow also doesn't reflect the process for core EIPs. I think updating the diagram would be a better idea than including this here.

Reply:
I have added the accepted status here https://github.com/fulldecent/EIPs/commit/c84ee64c6e97c3689ebd52f09cce4fd9159274e8

Last Call is good wording, implemented here https://github.com/fulldecent/EIPs/commit/d6bed1bc5454a241d9b482419274fb228d755f80

Swift Evolution uses Active Review, that would be fine too https://apple.github.io/swift-evolution/

The diagram is a nice to have. And the current options on the table are unnecessarily complicated. Please do not condition the acceptance of this PR on a diagram because currently the EIP has a huge problem, we can finalize any EIPS, and this PR is the fastest way to fix that.

Reply:
I'm not saying you *must* update the diagram - just that the ascii diagram you add here is overly simplified and thus likely to mislead.